src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -133,6 +133,14 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
             _state = RpcWorkerChannelState.Default;
         }
 
+        public Process Process
+        {
+            get
+            {
+                return (_rpcWorkerProcess as RpcWorkerProcess).Process;
```

Comment:
Should this be tied to the specific implementation? Can it be `_rpcWorkerProcess as WorkerProcess` ?

Reply:
I added `IWorkerProcess` to `IWorkerChannel` which is even better. Both `HttpWorkerChannel` and `GrpcWorkerChannel` can access to `IWorkerProcess` now.